Flexx/Classes/Utils/LocaleFileHandler.swift

```diff
@@ -124,6 +124,7 @@ class LocaleFileHandler {
      - returns: Data of that file
      */
     static func readLocaleFile(filename: String, domain: String = "") -> Data {
+        let languageCode = Flexx.shared.checkLocaleId(localeId: filename)
```

Comment:
Same question here: "Why we don't provide the language code as `fileName` param?"